flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/planner/expressions/ScalarFunctionsTest.scala

```diff
@@ -282,6 +282,13 @@ class ScalarFunctionsTest extends ScalarTypesTestBase {
       "NULL")
   }
 
+  @Test
+  def testTranslate(): Unit = {
+    testSqlApi("translate('AaBbCc', 'abc', '123')", "A1B2C3")
```

Comment:
Would be great to see also negative tests like with wrong amount of arguments or wrong argument types

Reply:
Add translate unit tests.